gbols/fast-food
# #160476499 Endpoint  to query a single order.

**What does this PR do?**
It gets a single order from the database.

**Description of Task to be completed?**
   - create an endpoint to place a new order.
   - create an endpoint to update status of an order.

**How should this be manually tested?**
   - pull from this branch `ft-get-single-order-160476499` into a new branch.
   - run `npm install` to install all the requirements.
   - endpoint should look like this `http://localhost:3000/api/v1/orders/:id.`
   - run `npm test` to run the test.

**What are the relevant pivotal tracker stories?**
#160476499 user should be able to get a single food order.

**Postman Screenshots**
![screenshot from 2018-09-13 09-33-38](https://user-images.githubusercontent.com/27724644/45478826-bb1d9900-b73c-11e8-9c73-4a657cccb060.png)
![screenshot from 2018-09-13 09-36-12](https://user-images.githubusercontent.com/27724644/45478839-c53f9780-b73c-11e8-8b77-974b44b58ea7.png)
